server/image_pull.go

```diff
@@ -203,6 +205,13 @@ func (s *Server) pullImage(ctx context.Context, pullArgs *pullArguments) (string
 					img, fmt.Sprintf("%d", imageSize(tmpImg)),
 				)
 
+				// Metrics for image pulls bytes
+				metrics.Instance().MetricImagePullsBytesAdd(
+					float64(p.OffsetUpdate),
+					p.Artifact.MediaType,
+					p.Artifact.Size,
+				)
+
 				// Metrics for size histogram
 				if p.Event == imageTypes.ProgressEventDone {
 					metrics.Instance().MetricImagePullsLayerSizeObserve(p.Artifact.Size)
```

Comment:
I forget, are we deprecating this one?

Reply:
Which one?
`ImagePullsLayerSizeObserve` we're not.

Reply:
doesn't this have the possibility of having high cardinality? (a different field for each distinct size)

Reply:
It's a Histogram with already sorted bucket sizes: so no harm to cardinality.
https://github.com/cri-o/cri-o/blob/5b7e1417fbb95bb8bda996bf96719700fbe4d06b/server/metrics/metrics.go#L172-L191

Reply:
reading is hard :upside_down_face: thank you